src/modules/dpos/validator/components/EditVote/index.js

```diff
@@ -209,8 +212,8 @@ const EditVote = ({ history, voteEdited, network, voting }) => {
             </>
           )}
         </BoxContent>
-        <BoxFooter direction={mode === 'edit' ? 'horizontal' : 'vertical'}>
-          {mode === 'edit' && (
+        <BoxFooter direction={mode === 'edit' && isForm ? 'horizontal' : 'vertical'}>
```

Comment:
retain either `mode`, or `isForm`.  they appear to perform a similar task.

Reply:
so `mode` performs the function of telling the component if its to display the edit content or the add content while `isForm` is used to know if it should render the confirmation modal or not (this is regardless if its in edit or add mode)